library/psa_crypto_core.h

```diff
@@ -50,13 +50,28 @@ typedef struct {
      * Number of locks on the key slot held by the library.
      *
      * This counter is incremented by one each time a library function
-     * retrieves through one of the dedicated internal API a pointer to the
-     * key slot.
+     * calls the psa_lock_key_slot() API, library functions must do this
+     * before they read the current content of the slot for an operation.
```

Comment:
(minor) "library functions must..." should begin a new sentence, and it should be "current contents"